Makefile

```diff
@@ -36,7 +36,7 @@ TARGET = -target aarch64
 else ifeq ($(ARCH),x86_64)
 CFLAGS += -D__ASM_SYSREG_H
 KERNEL_ARCH_NAME = x86
-KERNEL_HEADER_VERSION ?= 4.4.0-98-generic
+KERNEL_HEADER_VERSION ?= 4.11.0-14-generic
```

Comment:
should we bump the aarch64 one also? It's currently at 4.10.0-14

Reply:
Honestly I don't know much about the ARM build. @bjackson41 do you have any insights on this?

Reply:
4.10.0-14 was a minimum to get things working on aarch64. I can do some testing to see if this 4.11.0-14 can work as well.

Reply:
There were some build issues on ARM when trying to upgrade to 4.11. I made a ticket to track it but I think it might be something that should be addressed in a separate PR